### PR TITLE
created games controller

### PR DESCRIPTION
**Pull request by Young-Phoenix23**
Hey guys @Shouko0522 , @RenGVZ 
So I made the games controller and the show, new, index and edit. I tested it on the localhost and it works ok, but please take a look at it. My design is pretty terrible so I can change anything. I started working on the authorization, so that only an owner could create, edit or delete, but thought you guys should have a look before i moved on. I've been absent from class for a few weeks, so I'm sure I'm missing somethings. 
Thanx!!!

### CODE REVIEW

File: config/routes.rb [hunk 1]
```diff
@@ -1,5 +1,6 @@
 Rails.application.routes.draw do
   devise_for :users
   root to: 'pages#home'
+  resources :games
```
Comment (RenGVZ) [inline config/routes.rb]: I think something is going wrong here, that's why there's a conflict. Did you delete 'resources :reservations' from the original file?

**Comment by RenGVZ** [comment]
Hey Nii @Young-Phoenix23, I'm just checking it out now. Looks like you added some nice CSS. However I'm not sure git will let it merge because there are some conflicting files...

**Comment by RenGVZ** [comment]
Oh my bad! I didn't mean to close it lol still not used to how this thing works

File: config/routes.rb (hunk 1 above)
Comment (Young-Phoenix23) [reply config/routes.rb]: naw I think me and Shouko were working on the two controllers at the same time. Once she finished the file that I was working on wasn't the original file anymore.
Should i jus add resources :reservations to my branch?

Comment (Young-Phoenix23) [reply config/routes.rb]: I'll pull request it again

Comment (RenGVZ) [reply config/routes.rb]: That could be the problem, I'll look out for the new request

Comment (Young-Phoenix23) [reply config/routes.rb]: ok i think we are good to go!!

**Pull request merged**